node/pkg/txverifier/suitypes.go

```diff
@@ -130,34 +131,35 @@ func (o ObjectChange) ValidateTypeInformation(expectedPackageId string) (success
 // The response object for suix_tryMultiGetPastObjects
 type SuiTryMultiGetPastObjectsResponse struct {
 	SuiApiStandardResponse
-	Result []SuiTryMultiGetPastObjectsResult `json:"result"`
+	Result [2]SuiTryMultiGetPastObjectsResult `json:"result"`
 }
 
 // Gets the balance difference of the two result objects.
 func (r SuiTryMultiGetPastObjectsResponse) GetBalanceDiff() (*big.Int, error) {
 
 	if len(r.Result) != 2 {
-		return big.NewInt(0), fmt.Errorf("incorrect number of results received")
+		return big.NewInt(0), fmt.Errorf("incorrect number of objects in Result")
```

Comment:
Since you're using a pointer return type here, I think it's better to return `nil` in this function than a BigInt of 0.
```suggestion
		return nil, fmt.Errorf("incorrect number of objects in Result")
```